docs/api/vulnerability-info.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Vulnerability Info, NuGet API
+description: The data that allows clients to check packages for known vulnerabilities.
+author: zivkan
+ms.author: zivkan
+ms.date: 06/16/2023
+ms.topic: reference
+---
+
+# Vulnerability information
+
+NuGet client, starting from version 6.7, can download known package vulnerability information to use in scenarios such as checking packages during restore operations.
```

Comment:
Do you think it would be worth mentioning why this resource would be used over vulnerability info in the package metadata resource?

Reply:
Update looks good. Thanks.